doc/ch-image.rst

```diff
@@ -751,6 +751,12 @@ in other :code:`ARG` before the first :code:`FROM`.
 The :code:`FROM` instruction accepts option :code:`--arg=NAME=VALUE`, which
 serves the same purpose as the :code:`ARG` instruction. It can be repeated.
 
+:code:`LABEL`
+~~~~~~~~~~~~~
+
+The :code:`LABEL` instruction accepts :code:`key=value` pairs to
+add metadata for an image. Unlike Docker, multiline values are not supported. Can be repeated.
```

Comment:
1. Add a link to the issue to add multiline support (``(`issue #xxxx <https:/...>`_)``).
2. Wrap paragraph to 80 characters.

Reply:
Ok, should be all good now!